src/main/java/uk/gov/companieshouse/api/strikeoffobjections/model/model/Email.java

```diff
@@ -0,0 +1,92 @@
+package uk.gov.companieshouse.api.strikeoffobjections.model.model;
+
+import java.util.Map;
+
+public class Email {
```

Comment:
this should implement serializable

Reply:
https://github.com/companieshouse/promise-to-file-api/blob/master/src/main/java/uk/gov/companieshouse/api/promisetofile/model/email/EmailDocument.java#L3

You're right it probably should but the one in ptf doesn't

Reply:
It looks like something somewhere is being too lenient.

Reply:
I'll implement serializable here anyway